sms_statistics_service/src/firestore_wrapper.py

```diff
@@ -47,14 +62,23 @@ def update_sms_stats(self, project_name, sms_stats_batch):
         :type sms_stats_batch: dict of str -> src.data_models.SMSStats
         """
         log.info(f"Batch updating {len(sms_stats_batch)} SMS stats for project {project_name}...")
+        batch = self.client.batch()
+        batch_counter = 0
+        total_progress = 0
+        for iso_string, sms_stats in sms_stats_batch.items():
+            batch.set(self._get_sms_stat_doc_ref(project_name, iso_string), sms_stats.to_dict())
+            batch_counter += 1
+            total_progress += 1
 
-        doc_ref = self._get_sms_stats_doc_ref(project_name)
-        updates = {iso_string: sms_stats.to_dict() for iso_string, sms_stats in sms_stats_batch.items()}
+            if batch_counter >= self.MAX_BATCH_SIZE:
+                batch.commit()
+                log.info(f"Batch of {batch_counter} messages committed, progress: {total_progress} / {len(sms_stats_batch)}")
+                batch = self.client.batch()
```

Comment:
I don't know if you need to do this? I have a feeling you don't, could you check with one of e.g. the batch writing coda tools or the docs

Reply:
I did this because that's what the firestore_client_wrapper does in the Coda 2 repo.

But I've just tested experimentally*, and you're right, I don't need this.

(* by running with >500 items, which succeeds)